Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7
+
+COPY . /securitybot
+
+env PYTHONPATH $PYTHONPATH:/securitybot
+
+RUN apt-get update && \
+    apt-get install -y mysql-client && \
+    pip install -r /securitybot/requirements.txt
+
+WORKDIR /securitybot
+
+ENTRYPOINT ["/securitybot/docker_entrypoint.sh"]
```

Comment:
Per docker best practices, we should avoid running securitybot as root in the container. How about adding a new user, then adding: `USER securitybot`